app/src/main/java/net/skyscanner/backpack/demo/data/CalendarStoryType.kt

```diff
@@ -84,7 +84,7 @@ enum class CalendarStoryType {
                     selectionMode = rangeSelectionModeWithAccessibilityLabels(),
                     monthSelectionMode = CalendarParams.MonthSelectionMode.SelectWholeMonth(
                         label = "Select whole month",
-                        selectableMonthRange = YearMonth.now().plusMonths(1)..YearMonth.now().plusMonths(3),
+                        selectableMonthRange = YearMonth.of(2019, 1).plusMonths(1)..YearMonth.now().plusMonths(3),
```

Comment:
fixed selectable range to be within the pre defined interval in CalendarStory [here](https://github.com/Skyscanner/backpack-android/blob/917e34526471f38174b32e9c20ed8af65f680d15/app/src/main/java/net/skyscanner/backpack/demo/data/CalendarStoryType.kt#L36-L37)